### PR TITLE
Update CDA script and improve debugging output

### PR DESCRIPTION
**Pull request by jeanconn**
Update CDA script and improve debugging output.

Current failures in the cron job aren't reproducible at the cmd line and there isn't enough output from the cron job failures to see what is going on.  This PR should at least close the hdf5 file properly and might give enough debugging output to see  if there is a problem with the source data from the url.

### CODE REVIEW

**Comment by jeanconn** [comment]
I think I can run this on the side for a bit and confirm that the table looks the same as the official table.

File: mica/archive/cda_aca0_list.py [hunk 1]
```diff
@@ -50,11 +49,11 @@ def make_data_table(lines):
     filetimes = [int(file_re.search(f).group(1)) for f in files['filename']]
     versions = [int(file_re.search(f).group(2)) for f in files['filename']]
     now_dates = np.repeat(DateTime().date, len(ingest_dates))
-    files = Ska.Numpy.add_column(files, 'ingest_date', ingest_dates)
-    files = Ska.Numpy.add_column(files, 'aca_ingest', now_dates)
-    files = Ska.Numpy.add_column(files, 'filetime', filetimes)
-    files = Ska.Numpy.add_column(files, 'version', versions)
-    files.sort(order=['aca_ingest', 'ingest_date', 'filename'])
+    files.add_column(Column(ingest_dates, 'ingest_date'))
```
Comment (taldcroft) [inline mica/archive/cda_aca0_list.py]: `files['ingest_date'] = ingest_dates` is the recommended idiom now.

**Comment by taldcroft** [comment]
While you are at it, a module docstring saying what this does would be good.  I don't know with certainty what CDA even stands for (Chandra Data Archive would be my guess), and I guess this relates to some custom web query script that arcops made for you?

**Comment by jeanconn** [comment]
> While you are at it, a module docstring saying what this does would be
> good. I don't know with certainty what CDA even stands for (Chandra Data
> Archive would be my guess), and I guess this relates to some custom web
> query script that arcops made for you?
>
​I don't know if I'd call it "custom".  it was intended to be a generic
interface to get the list of Chandra Data Archive files of various types by
date so that we could then do arc45gl queries by filename or some such:

​https://icxc.harvard.edu/dbtm/CDA/aspect_fetch.html

I've been using the aca0 list of files to see if we've got everything in
the mica version of the archive.

File: mica/archive/cda_aca0_list.py [hunk 2]
```diff
@@ -36,8 +35,8 @@ def get_options():
 
 
 def make_data_table(lines):
-    files = asciitable.read(lines,
-                            names=['filename', 'status', 'ingest_time'])
+    files = Table.read(lines, format='ascii.csv',
```
Comment (jeanconn) [inline mica/archive/cda_aca0_list.py]: Add guess=false per @taldcroft

**Comment by jeanconn** [comment]
It looks like the astropy.table changes I made aren't completely compatible with the manual pytables hdf5 row appending this code was doing.  @taldcroft do you have any example code using the astropy.table hdf5 support to add rows to an hdf5 table?  Without rewriting the whole thing?

**Comment by taldcroft** [comment]
Astropy `Table` does not have direct backend support for hdf5 row manipulation.  In general Table serves just as a container and not an active database proxy.  Blaze might do this, but it's an heroic undertaking.  Sorry.

**Comment by jeanconn** [comment]
Good to know.  Thanks @taldcroft

**Comment by jeanconn** [comment]
It looks like the change in  d12e9c6 was really all I needed to make this work again.  Lately this seems to be timing out when run as a cron task, but I'm not seeing that when running manually.  This change should just log the URL or HTTP errors instead of exiting and also includes other minor fixes and debugging content.

**Comment by taldcroft** [comment]
I don't think I can meaningfully comment on this now, so don't wait for me.

**Pull request merged**